.gitignore

```diff
@@ -25,10 +22,9 @@ logs/
 
 # Other
 **/.factorypath
-**/developer.properties
```

Comment:
In the current version of Qube these are not ignored, why is this preferred? I thought we wanted to avoid uploading sensitive information?

Reply:
Oha, then please open an issue on QUBE

Reply:
Done! https://github.com/qbicsoftware/qube-cli/issues/181

Reply:
Done! (Feel free to add it manually here).